pyproject.toml

```diff
@@ -160,7 +160,13 @@ free-threaded-support = true
 before-build = "PACKAGE_DIR={package} bash {package}/scripts/cibw_before_build.sh"
 
 [tool.cibuildwheel.windows]
-before-build = "pip install delvewheel && bash {package}/scripts/cibw_before_build.sh"
+before-build = "pip install delvewheel && bash {package}/scripts/cibw_before_build_windows.sh"
+before-test = "bash {package}/scripts/cibw_before_test_windows.sh"
+test-command = """
+  set PANDAS_CI='1' && \
```

Comment:
Could the `PANDAS_CI` be set in the `before-test` so that this section can re-use the `test-command`?

Reply:
We could do that, yes, but all `cibuildwheel` custom configs include a different `test-command`. The Windows one has a slight difference as well.